aten/src/THC/generic/THCTensorMathPointwise.cu

```diff
@@ -115,22 +115,6 @@ void THCTensor_(cross)(THCState *state, THCTensor *self, THCTensor *x, THCTensor
 {
   THCAssertSameGPU(THCTensor_(checkGPU)(state, 3, self, x, y));
 
-  int i;
```

Comment:
you should probably rename this function to crossKernel or something because you got rid of the checking.  Otherwise someone from THC might call this thinking it's a proper function.

Reply:
Renamed